src/OpenApi/Model/Paths.php

```diff
@@ -27,7 +27,7 @@ public function getPath(string $path): ?PathItem
         return $this->paths[$path] ?? null;
     }
 
-    public function getPaths(): array
+    public function getPaths(): ?array
     {
         return $this->paths;
```

Comment:
Isn't it possible to initialize the property to an empty array instead?

Reply:
Sure 👌